subsys/rtk/protocol/rtcm3_crc.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
I'd like some feedback WRT this file: As far as I'm concerned this CRC calculation is specific to RTCM3, however I don't know if it's preferred to put this under the more generic `include/zephyr/sys/crc.h`

Reply:
Moved to `zephyr/sys/crc.h`